### DISCUSSION TITLE
Welcome to edge-tts Discussions!

### DISCUSSION THREAD

**Discussion by rany2** [rany2/edge-tts#80] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jesuiskiki** [comment]
Does this support Chinese？

**Reply by rany2** [reply]
Yes, you could use a Chinese voice. Check the README or examples for how to do that but the subtitles are still a problem which is being worked on.
